media/js/base/fxa-utm-referral.js

```diff
@@ -78,24 +75,29 @@
         }
 
         for (var i = 0; i < ctaLinks.length; i++) {
-            // get the current FxA links.
+            // get the link off the element
             var oldAccountsLink = ctaLinks[i].href;
-            var oldMozillaOnlineLink = ctaLinks[i].getAttribute('data-mozillaonline-link');
-
-            // append our new UTM param data to create new FxA links.
-            var newAccountsLink = UtmUrl.appendToDownloadURL(oldAccountsLink, params);
-
-            // set the FxA button to use the new link.
-            ctaLinks[i].href = newAccountsLink;
-
-            // also handle mozilla-online links for FxA China Repack.
-            if (oldMozillaOnlineLink) {
-                var newMozillaOnlineLink = UtmUrl.appendToDownloadURL(oldMozillaOnlineLink, params);
-                ctaLinks[i].setAttribute('data-mozillaonline-link', newMozillaOnlineLink);
+            // verify this a link to accounts.firefox.com or dev server, otherwise we shouldn't touch it
+            if(oldAccountsLink.indexOf('https://accounts.firefox.com') === 0 || oldAccountsLink.indexOf('https://latest.dev.lcip.org') === 0) {
```

Comment:
Thank you for carefully going back through the comments in the old PR and picking out this piece of code that was omitted 🙏